File: systems-for-design-and-implementation/labs/lab3/LibraryNetworking/src/library/network/dto/TupleInt.scala
package library.network.dto

import java.io.Serializable

// TODO use scala tuples. sigh
class TupleInt(var first: Int, var second: Int) extends Serializable
{
    def getFirst: Int =
    {
        first
    }

    def setFirst(first: Int)
    {
        this.first = first
    }

    def getSecond: Int =
    {
        second
    }

    def setSecond(second: Int)
    {
        this.second = second
    }

    override def toString: String =
    {
        "TupleInt{" + "first=" + first + ", second=" + second + '}'
    }
}

File: systems-for-design-and-implementation/labs/exam/TicTacToeNetworking/src/tictactoe/network/dto/TupleInt.java
package tictactoe.network.dto;

import java.io.Serializable;


public class TupleInt implements Serializable
{
    private int first;
    private int second;

    public TupleInt(int first, int second)
    {
        this.first = first;
        this.second = second;
    }

    public int getFirst()
    {
        return first;
    }

    public void setFirst(int first)
    {
        this.first = first;
    }

    public int getSecond()
    {
        return second;
    }

    public void setSecond(int second)
    {
        this.second = second;
    }

    @Override
    public String toString()
    {
        return "TupleInt{" +
            "first=" + first +
            ", second=" + second +
            '}';
    }
}
